rdf-star-cg-spec.html

```diff
@@ -1034,7 +1058,7 @@ <h2>Interpolation lemma</h2>
   <section class="appendix">
     <h2>Historical remarks</h2>
 
-    <section class="appendix">
+    <section>
       <h2>SA-mode and PG-mode</h2>
```

Comment:
```suggestion
      <h3>SA-mode and PG-mode</h3>
```

Reply:
From https://respec.org/docs/:
> Using h2 everywhere is sort of a tacit convention

plus, Respec changes the Hx tags itself into a consistent hierarchy.
Finally, even with H2 everywhere, this is still valid HTML...

Reply:
> > Using h2 everywhere is sort of a tacit convention

Ugh.  Respect is basically saying, "Be lazy and sloppy, authors!  We'll fix it in post!"

But I guess your dialect has now been blessed.

> plus, Respec changes the Hx tags itself into a consistent hierarchy.

*IFF* everything is properly wrapped and nested in `<section>` tags.  An important distinction without which the auto-hierarchy-construction will fail.

> Finally, even with H2 everywhere, this is still valid HTML...

I never said it wasn't valid HTML.  Plenty of valid HTML leads to incomprehensible browser rendering.  I'm glad that doesn't happen here.  I still believe it's best to keep the H# tags in the source numbered as they're intended to be rendered.

Reply:
> Ugh. Respect is basically saying, "Be lazy and sloppy, authors! We'll fix it in post!"

I look at it more positively, as "be careful to nest your sections correctly, and I'll take care of the tedious redundant numbering of H's" ;-)

> IFF everything is properly wrapped and nested in <section> tags. 

Of course, but my personal opinion is that <section>`s aren't optional nowadays. I think HTML would have been better off from the start with a `<section>` tag and a single `<h>` tag ;-)

Reply:
@pchampin - I cannot fully decipher your comment above, and neither can Github (as I cannot "quote reply" at all).  Can you correct the markdown therein, so it renders as intended?

Reply:
Sorry, sent the previous comment too hastily :confused: This is now fixed.

Reply:
@pchampin - I'm sure I'm failing to mentally fill in something that should be obvious, but I'm even more sure that the markup is not leading to rendering as intended. Perhaps something was eaten/deleted when you saved it. As it stands, I think there's an extra backtick preceding `sections aren't optional` and some further imbalance as you go, with impact following `start with a`.

Reply:
:fearful: sorry again... should be good now

Reply:
*shrugs*  I think we must agree to disagree.  I prefer to keep the source as close as possible to the desired outcome, partly because it limits confusion when others review that source... which is where I always start, because the changes are highlighted better here than in the Respec DIFF.  Back to focusing on the content instead of the markup.  :-)